docs/specification.yml

```diff
@@ -2842,7 +2842,7 @@ components:
             type: object
           type: array
           uniqueItems: true
-    DAC:
+    REMS_DAC:
```

Comment:
this is a workflow in REMS not a dac, maybe we can keep terminology, because from REMS workflow we derive a the  DAC metadata specific to the EGA model

Reply:
It is called `REMS DAC` in the submission schema. Perhaps a new issue for that?

Reply:
Mmm, I was also wondering what constitutes as "DAC" and just "workflow". As @csc-felipe mentioned, `submission.json` originally titled these as `"REMS DAC for datasets"` but perhaps this was inaccurate titling? Should all of these just be `REMS workflows` instead?

Reply:
new issue sounds fine.